<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Alert</class>
 <widget class="QWidget" name="Alert">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>162</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>162</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>162</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Message</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/SVG/svg/icon/audiobook-editor-logo.svg</normaloff>:/SVG/svg/icon/audiobook-editor-logo.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #474747;
box-shadow: 0px 4px 15px rgba(0, 0, 0, 0.28);
border-radius: 6px;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>9</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans TC</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/*
font-family: Noto Sans;
font-style: normal;
font-weight: 600;
font-size: 12px;
line-height: 16px;
display: flex;
align-items: center;
*/
/* panel/section title */

color: rgba(255, 255, 255, 0.5);</string>
   </property>
   <property name="text">
    <string>Alert</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_Close">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>5</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/SVG/svg/icon/attach-panel-close.svg</normaloff>:/SVG/svg/icon/attach-panel-close.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_Msg">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>34</y>
     <width>247</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans TC</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/*
font-family: Noto Sans;
font-style: normal;
font-weight: normal;
font-size: 13px;
line-height: 18px;
*/
color: rgba(255, 255, 255, 0.65);</string>
   </property>
   <property name="text">
    <string>You must select one directory and keyin the booktitle! You must select one directory and keyin the booktitle!</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Ok">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>124</y>
     <width>247</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans TC</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
align-items: center;

background: #3095F2;
border-radius: 5px;
/*
font-family: Noto Sans;
font-style: normal;
font-weight: normal;
font-size: 13px;
line-height: 18px;
*/
text-align: center;
color: #FFFFFF;</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
